README.md

```diff
@@ -24,17 +23,51 @@ This Kubernetes-based version is built using the same approach as the [machine c
 a situation where your Kubernetes cluster and its ingress controllers have
 a fast connection to the Internet.
 
+For information about how to deploy, integrate, and manage this charm, see the Official [Content Cache Documentation](https://charmhub.io/content-cache-k8s/docs).
+
+## Get Started
+
+To begin, refer to the [Getting Started](https://charmhub.io/content-cache-k8s/docs/tutorial-getting-started) tutorial for step-by-step instructions.
```

Comment:
```suggestion
To begin, refer to the [Content Cache tutorial](https://charmhub.io/content-cache-k8s/docs/tutorial-getting-started) for step-by-step instructions.
```
We've been moving away from referring to our tutorials as "getting started" or "quick guide", although I see that this tutorial has not been updated yet.

Reply:
addressed